--- a/.bashrc	2020-01-16 16:49:20.590213977 -0600
+++ b/.bashrc	2020-01-16 16:56:32.927892810 -0600
@@ -111,3 +111,16 @@
     . /etc/bash_completion
   fi
 fi
+
+# if this is the first time a user has logged in
+if [ ! -f /spacepi/build/Makefile ]; then
+    # remount everything read-write
+    sudo mount -o remount,rw /
+    sudo mount -o remount,rw /boot
+    sudo mount -o remount,rw /spacepi/bin
+    mount /spacepi/code
+    mount /spacepi/build
+
+    # fix permissions
+    sudo chmod -R g+w /spacepi/code /spacepi/build /spacepi/bin
+fi
