--- a/hwclock.sh	2020-01-15 16:51:44.551414354 -0600
+++ b/hwclock.sh	2020-01-15 16:52:11.979181637 -0600
@@ -55,7 +55,7 @@
 	start)
 	    # If the admin deleted the hwclock config, create a blank
 	    # template with the defaults.
-	    if [ -w /etc ] && [ ! -f /etc/adjtime ] && [ ! -e /etc/adjtime ]; then
+	    if [ -w /etc ] && [ ! -L /etc/adjtime ] && [ ! -e /etc/adjtime ]; then
 	        printf "0.0 0 0.0\n0\nUTC\n" > /etc/adjtime
 	    fi
 
